packages/core-typings/src/IMessage/MessageAttachment/MessageAttachmentBase.ts

```diff
@@ -21,3 +21,14 @@ export type MessageAttachmentBase = {
 		sha256: string;
 	};
 };
+
+export type EncryptedMessageAttachment = MessageAttachmentBase & {
+	encryption: {
+		iv: string;
+		key: JsonWebKey;
+	};
```

Comment:
```suggestion
	encryption: Required<MessageAttachmentBase>['encryption'];
```

Either this or create a type `AttachmentEncryption`.